src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -34,37 +41,66 @@ public String greeting(@RequestParam(name="name", required=false, defaultValue="
     @Autowired
     private BookRepository bookRepository;
 
+    /**
+     * Method that find all the information of one entity.
+     * @return
```

Comment:
Indicate what is returned after the @return tag. 
Apply this on every empty return

Reply:
I added javadoc in tag retun